src/content/en/updates/2017/09/sensors-for-the-web.md

```diff
@@ -0,0 +1,410 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Generic Sensor API is available for Origin Trials in Chome 62.
+
+{# wf_updated_on: 2017-09-20 #}
+{# wf_published_on: 2017-09-20 #}
+{# wf_tags: sensors,origintrials,chrome62,news #}
+{# wf_blink_components: Blink>Sensor #}
+{# wf_featured_image: /web/updates/images/generic/sd-card.png #}
+{# wf_featured_snippet: Generic Sensor API is available for Origin Trials in Chome 62. #}
+
+# Sensors For The Web! {: .page-title }
+
+{% include "web/_shared/contributors/alexshalamov.html" %}
+{% include "web/_shared/contributors/pozdnyakov.html" %}
+
+Today, sensor data is used in many native applications to enable use cases such
+as immersive gaming, fitness tracking, and augmented or virtual reality. Wouldn't
+it be cool to bridge the gap between native and web applications? The [Generic Sensor
+API](https://www.w3.org/TR/generic-sensor/), FTW!
+
+## What is Generic Sensor API? {: #what-is-generic-sensor-api }
+
+The [Generic Sensor API](https://www.w3.org/TR/generic-sensor/) is a set of
+interfaces which expose sensor devices to the web platform. The API consists
+of the base [Sensor](https://w3c.github.io/sensors/#the-sensor-interface)
+ interface and a set of concrete sensor classes built on top. Having a base
```

Comment:
remove extra space at beginning of line, it'll make the MD processor angry... and throughout rest of document